pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -438,3 +438,8 @@ def test_parse_latex_table_wrapping(styler):
         overwrite=False,
     )
     assert _parse_latex_table_wrapping(styler.table_styles, None) is True
+
+
+def test_long_captions(styler):
+    result = styler.to_latex(caption=("short cap", "long cap"))
+    assert "\\caption[short cap]{long cap}" in result
```

Comment:
Can you replicate the existing behavior as here https://github.com/pandas-dev/pandas/pull/35668/files?
Currently in ``DataFrame.to_latex`` we can pass a string ``full_caption`` or a tuple ``(full_caption, short_caption)``.
In your code it is vice versa ``(short_caption, full_caption)``.
Also I would rename this test function to ``test_short_caption``, since it targets a feature of adding a short caption.
In addition, you would need to change the indices in the docstring as well.

Reply:
good catch. easy fix. should be ok now.